lib/_debug_agent.js

```diff
@@ -102,11 +106,17 @@ function Client(agent, socket) {
   this.socket.on('close', function() {
     self.destroy();
   });
+  this.socket.on('error', function(err) {
+    // Silently handle socket errors
```

Comment:
Also this is a place for discussion. I don't know how we need to handle debugger connection errors (My opinion is - we don't need to do anything on debugger connection errors, because this is debugging of debugger)